docs/general-information/project-template.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+title: Project Template
+parent: General Information
+---
+
+# Project Template
+
+## Overview
+
+The project template is a base project with all the basic functionality built in for developing
+Python or R projects. This helps save time and effort at the beginning of a project and keeps
+project development workflows and standards consistent across the lab. The base project uses
+[Docker](https://ndclab.github.io/wiki/docs/technical-docs/docker-usage.html) as a development
+platform and as a deployment platform.
+
+By default the project contains a project directory, markdown templates for issues and pull
+requests, shell scripts to automate some basic actions, and yaml configurations for [GitHub
+Actions].
+
+## Usage
+
+To use the template first clone the project template
+[repository](https://github.com/NDCLab/project-template/tree/main).
```

Comment:
Maybe have a sentence after this part saying something like" "See [github main doc] for a description of how to clone a repo"

Reply:
I do NOT think you need to repeatedly refer to other docs throughout this doc, but maybe just the first time you refer to a set of concepts covered in another doc?